setup.py

```diff
@@ -7,20 +7,22 @@
 extras_require = {
     "dev": [
         "build>=0.9.0",
-        "bumpversion>=0.5.3",
+        "bump_my_version>=0.19.0",
         "ipython",
+        "mypy==1.10.0",
```

Comment:
If mypy is disabled, maybe we should exclude it from here for now?

Reply:
removed!